src/js/player.js

```diff
@@ -1017,6 +1022,13 @@ class Player extends Component {
     this.tech_.dispose();
 
     this.tech_ = false;
+
+    if (this.isPosterFromTech_) {
```

Comment:
unset the poster if the poster was set via the tech? Interesting. That sounds like a great idea.